### PR TITLE
fix(DeskMentions): Fix the problem where desk mentioned query was called recursively.

### PR DESCRIPTION
**Pull request by mdhaman**

### CODE REVIEW

File: scripts/superdesk-monitoring/monitoring.js [hunk 1]
```diff
@@ -749,7 +752,7 @@
                  */
                 scope.acknowledge = function(notification) {
                     deskNotifications.markAsRead(notification, scope.desk);
-                    $timeout(init, 5000);
+                    $timeout(reload(), 5000);
```
Comment (petrjasek) [inline scripts/superdesk-monitoring/monitoring.js]: I guess it should be only `reload` here, without `()`

Comment (mdhaman) [inline scripts/superdesk-monitoring/monitoring.js]: yes.

**Comment by petrjasek** [comment]
this PR should fix travis https://github.com/superdesk/superdesk-client-core/pull/891

**Comment by mdhaman** [comment]
thanks @petrjasek

**Comment by akintolga** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**